docs/source/examples/maxcut-demo.myst

```diff
@@ -0,0 +1,481 @@
+---
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.10.2
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# Solving MaxCut with QAOA improved by zero-noise extrapolation
```

Comment:
Maybe make the title say improved with Mitiq?  Obv that's the technique you are using, but then is more clear you are using Mitiq to do so :)